client/components/LearnerViewDirective.js

```diff
@@ -239,6 +318,7 @@ tie.directive('learnerView', [function() {
           $scope.previousInstructions = [];
           $scope.nextButtonIsShown = false;
           $scope.feedbackParagraphs = introParagraphs;
+          $scope.unlockedQuestions[$scope.currentQuestionIndex] = true;
```

Comment:
My first thought was -- but it's an array!

Then my next thought was -- but JS arrays are objects behind the scenes, so this does kind of work.

Still... what do you think of filling the array with `false` to start with? Then setting it makes more sense conceptually. I do think there is a conceptual difference between false and undefined.